docs/variants/protectli_v1000-series/releases.md

```diff
@@ -18,3 +18,74 @@ Test results for this platform can be found
 
 <!--Empty pixel to avoid orphaned pages when overview is hidden-->
 [![empty-pixel](../../images/empty_pixel.png)](overview.md)
+
+## Protectli V1210 Dasharo Release Notes
+
+Following Release Notes describe status of open-source firmware development for
+Protectli V1210
+
+For details about our release process please read
+[Dasharo Standard Release Process](../../dev-proc/standard-release-process.md).
+
+<center>
+
+[Subscribe to Protectli V1210 Dasharo Release Newsletter]
+[newsletter]{ .md-button .md-button--primary .center }
+
+</center>
+
+### v0.9.3 - 2024-09-09
+
+Test results for this release can be found
+[here](https://docs.google.com/spreadsheets/d/1wSE6xA3K3nXewwLn5lV39_2wZL1kg5AkGb4mvmG3bwE/edit?gid=1207991922#gid=1207991922).
+
+#### Added
+
+- Intel-specific HDA verbs, for proper audio functionality
+
+#### Changed
+
+- Disabled DSP
+- Disabled SATA due to lack of HW support
+- Disabled SSID programming to prevent Windows default drives from not probing
+  successfully
+- Removed differences in SMBIOS fields compared to proprietary FW
+- Disabled WiFi L0 to prevent errors from showing up in dmesg
+
+#### Known Issues
+
+- There is no PC speaker populated on the platform, so it does not give sound
+  signals on errors and boot
+- The chassis is getting very hot
+
+#### Binaries
+
+[protectli_v1210_v0.9.3.rom][protectli_v1210_v0.9.3.rom_file]{.md-button}
+[sha256][protectli_v1210_v0.9.3.rom_hash]{.md-button}
+[sha256.sig][protectli_v1210_v0.9.3.rom_sig]{.md-button}
+
```

Comment:
Since release notes and components are the same for V1410, we may just add v1410 links to binaries here and that's all